backend/danswer/server/documents/models.py

```diff
@@ -201,11 +224,23 @@ class CCPairFullInfo(BaseModel):
     def from_models(
         cls,
         cc_pair_model: ConnectorCredentialPair,
-        index_attempt_models: list[IndexAttempt],
         latest_deletion_attempt: DeletionAttemptSnapshot | None,
+        number_of_index_attempts: int,
+        last_index_attempt: IndexAttempt | None,
         num_docs_indexed: int,  # not ideal, but this must be computed separately
         is_editable_for_current_user: bool,
     ) -> "CCPairFullInfo":
+        # figure out if we need to artificially deflate the number of docs indexed.
```

Comment:
This logic was originally in the front end but i thought it made more sense to be here